devtools/devtools/behave/temporarydirectory.py

```diff
@@ -14,8 +18,20 @@ def get_context_temp_dir_path(context) -> Path:
 
         context.add_cleanup(lambda: context.temp_dir.cleanup())
 
-        context.add_cleanup(
-            lambda: shutil.move(context.temp_dir.name, "/root/last_run_behave")
-        )
+        if "KEEP_FILES" in os.environ and bool(distutils.util.strtobool(os.environ["KEEP_FILES"])):
```

Comment:
@canwaf - I've hidden your functionality for keeping the files behind here and tidied it up a bit so you don't end up with all those `tmp` folders building up in there.